config/environments/local.rb

```diff
@@ -22,8 +22,8 @@
   # Only use best-standards-support built into browsers
   config.action_dispatch.best_standards_support = :builtin
 
-  # Turn off asset pipline information showing in logs 
-  config.assets.logger = false 
+  # # Turn off asset pipline information showing in logs 
+  # config.assets.logger = true 
```

Comment:
Let's bring this line back, and set it to false?

Reply:
So I added this line originally. I can set it back to false but I believe it should be commented out. Are you ok with that?